editor/editor_export_godot3.cpp

```diff
@@ -1781,11 +1984,14 @@ Error EditorExportGodot3::export_godot3(const String &p_path) {
 	for (List<String>::Element *E = files.front(); E; E = E->next()) {
 
 		String file = E->get();
+		String file_local = file.replace("res://", "");
```

Comment:
This is because otherwise strings such as `pong.xml` wouldn't be converted in `godot.cfg`, even if the actual resource has been converted to `pong.tscn`.